Module_5/Q4.js

```diff
@@ -0,0 +1,4 @@
+const JsonString = '{"name": "sakshi", "age": 21, "city": "Solapur"}';
+const parse = JSON.parse(JsonString);
+
+console.log(parse);
```

Comment:
Have you tried passing some dirty data to Json.parse? What do we get ? What's the error? You can search online how should we handle such scenarios when there are chances that data is dirty or not in proper format which is passed to JSON.parse